### PR TITLE
DEV056 Invite user to organization

### PR DESCRIPTION
**Pull request by Maxxx911**
[card](https://trello.com/c/EofXhGMJ)

### CODE REVIEW

File: app/controllers/organizations_controller.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class OrganizationsController < ApplicationController
   authorize_resource
-  before_action :set_organization, only: %i[show update destroy edit]
+  before_action :set_organization, only: %i[show update destroy edit confirmation leave]
```
Comment (longbeardmen) [inline app/controllers/organizations_controller.rb]: action should be a verb like `confirm`

File: app/controllers/organizations_controller.rb [hunk 2]
```diff
@@ -36,7 +41,8 @@ def create
   end
 
   def edit
-    @users = User.all
+    @users = User.all.where.not(id: @organization.users.pluck(:id))
```
Comment (andreydro) [inline app/controllers/organizations_controller.rb]: You don't need method `all` in here

File: app/views/user_mailer/delete_email.html.erb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1>Welcome to example.com,</h1>
+    <p>
+      <%= @organization.name %>
+      You have successfully signed up to example.com,
```
Comment (andreydro) [inline app/views/user_mailer/delete_email.html.erb]: Maybe here should be email with context about deletion?

Comment (andreydro) [reply app/views/user_mailer/delete_email.html.erb]: Let's do it in Russian?

File: app/views/user_mailer/invite_email.html.erb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1>Welcome to example.com,</h1>
+    <p>
+      <%= @organization.name %>
+      <%= @user.email %>
+      You have successfully signed up to example.com,
```
Comment (andreydro) [inline app/views/user_mailer/invite_email.html.erb]: Let's write email in Russian?

Comment (Maxxx911) [reply app/views/user_mailer/invite_email.html.erb]: I just prepared email and Masha will do it beautifully

**Pull request merged**